src/backend/test/utils/invalid-upgrade-candidates/v100PR_v100PR2.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
suggested filename: `RC-sameVN-sameVC.json` e.g. two release candidates with the same version name and same version code. Invalid candidate because RC should only update if version code is higher.